_config.yml

```diff
@@ -0,0 +1,10 @@
+site:
+  url: https://openapi.tools
+description: A site to find the right tools for developers to build with the OpenAPI  
+collections_dir: /reviews
```

Comment:
Consider changing to something like `content` to allow different types of collections....